ModelViewer/ModelViewer/SettingsMenu.cpp

```diff
@@ -0,0 +1,220 @@
+#include "SettingsMenu.h"
+#include "ViewerGraphicsWindow.h"
+#include "TreeItem.h"
+#include "ModelViewer.h"
+
+#include <QGridLayout>
+#include <QListWidget>
+#include <QLabel>
+#include <qstringlist.h>
+#include <QStandardItemModel>
+#include <QFile>
+#include <QFormLayout>
+#include <QLineEdit>
+#include <QPushButton>
+
+
+SettingsMenu::SettingsMenu(QWidget* parent)
+	: QWidget(parent)
+{
+	setWindowTitle(QString::fromLatin1("Settings"));
+	setContentsMargins(0, 0, 0, 0);
+	resize(400, 300);
+
+	m_pMainLayout = new QGridLayout(this);
+
+	// Set up the settings
+	settings = new QSettings("TBD", "Model Viewer");	
```

Comment:
I think QSettings is static, so you don't have to pass around a pointer to the settings object.
You can just write `QSettings settings;` wherever you need access to the settings.  You still need to call `QSettings settings("TBD", "Model Viewer")` sometime at startup to initialize the app name: https://doc.qt.io/qt-5/qsettings.html#QSettings-4